baseline.rs

```diff
@@ -20,7 +20,7 @@ pub fn main() {
     let x = kc_borrow_mut!(val); // x = &mut val;
     let x_alias = x as *mut u8;
     let y = kc_borrow_mut!(*x);
-
+    test_dependencies::print_tag_of(b"y\0".as_ptr(), y);
```

Comment:
I think we need to bless the tests after this change? I would have thought CI wouldn't have passed without that...

Reply:
We don't validate the output from baseline.rs, its not part of the `kc/` stuff. Its just a "human eyeball" demo, for better or for worse.

Reply:
Oh, I got it confused with the files inside of `kc`